### PR TITLE
Docs: add indeterminate disabled checkbox example

### PR DESCRIPTION
**Pull request by julien-deramond**
Based on the work of @louismaximepiton this PR proposes to add a "Disabled indeterminate checkbox" example in **Forms > Check & radios > Disabled**.

### [Live preview](https://deploy-preview-36674--twbs-bootstrap.netlify.app/docs/5.2/forms/checks-radios/#disabled)

### CODE REVIEW

File: site/assets/js/snippets.js [hunk 1]
```diff
@@ -102,7 +102,9 @@
   // Indeterminate checkbox example in docs and StackBlitz
   document.querySelectorAll('.bd-example-indeterminate [type="checkbox"]')
     .forEach(checkbox => {
-      checkbox.indeterminate = true
+      if (checkbox.id.includes('Indeterminate')) {
```
Comment (julien-deramond) [inline site/assets/js/snippets.js:105]: The other way around would be to set `checkbox.indeterminate = true` only to the first example in `.bd-example-indeterminate`:
```js
document.querySelectorAll('.bd-example-indeterminate')
  .forEach(example => {
    example.querySelector('[type="checkbox"]').indeterminate = true
  })
```
IDK what's the preferred approach here.

**Comment by mdo** [comment]
Would we ever want to add a custom `indeterminate` attribute to use as a selector? If not, I think a class might make most sense.

**Comment by julien-deramond** [comment]
> Would we ever want to add a custom `indeterminate` attribute to use as a selector? If not, I think a class might make most sense.

You mean in the library itself (by having let's say as an example an `.indeterminate` class that will automatically launch the JS and then remove the class) or just for the documentation?

For the documentation I tried to avoid having an extra-class so that it is still understandable for the user (when copy/pasting or editing via StackBlitz) that the indeterminate state can only be set by JavaScript.

**Comment by mdo** [comment]
That all sounds good. I appreciate the goal of keeping the source code lean for the end user.

**Review by mdo — APPROVED** [review]

**Pull request merged**